CHANGELOG.md

```diff
@@ -16,7 +16,7 @@
 
 ### Changed
 
-* Update minimum supported rust to 1.69 for CVE-2020-26235
+* Update minimum supported rust to 1.70 for CVE-2020-26235
```

Comment:
Undo, looks like an accidental change when updating MSRV in all locations. Good job getting all the others though!